pkgs/build-support/build-sandbox/src/setup.c

```diff
@@ -680,6 +684,9 @@ static bool setup_static_etc(struct query_state *qs)
     return mount_requisites(qs, dest);
 }
 
+/* Query the nix daemon for the the requisites (transitive closure)
+   of ? TODO and the paths in mount_runtime_path_vars. All requisites
+   are bind-mounted into the /nix/store in the sandbox. */
```

Comment:
Well, the actual store path queries are not here but in `setup_static_etc` and `mount_runtime_path_vars`, here we just initialise and wrap the query state for these functions.